<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnooperDialogBase</class>
 <widget class="QDialog" name="SnooperDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snooper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>This tool helps developers and designers discover objects.
Click on anything anywhere to highlight an item.
Installs detroyed() handlers for all objects. When open, may slow program slightly.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="onlyAppCheckBox">
       <property name="text">
        <string>Only MusEGui</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onlyWidgetCheckBox">
       <property name="text">
        <string>Only QWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="separateParentedTopLevelsCheckBox">
       <property name="text">
        <string>Separate parented top levels</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="onlyPropsCheckBox">
       <property name="text">
        <string>Only properties</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update tree</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search class name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="classNameLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Search object name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="objectNameLineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="objectTree">
     <column>
      <property name="text">
       <string>Class name::Object name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Property</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
